https://bugzilla.redhat.com/show_bug.cgi?id=1262457

--- origsrc/kbruch-15.04.3/src/org.kde.kbruch.desktop	2015-04-27 06:12:22.000000000 -0500
+++ src/kbruch-15.04.3/src/org.kde.kbruch.desktop	2015-09-11 13:26:20.697583700 -0500
@@ -1,7 +1,7 @@
 # KDE Config File
 [Desktop Entry]
 Type=Application
-Exec=kbruch -caption %c %i
+Exec=kbruch -qwindowtitle %c
 Icon=kbruch
 X-DocPath=kbruch/index.html
 GenericName=Exercise Fractions
